cmd/ndm_daemonset/app/command/device-list.go

```diff
@@ -81,7 +81,6 @@ func NewSubCmdListBlockDevice() *cobra.Command {
 
 	// Bind & parse flags defined by external projects.
```

Comment:
These lines can also be removed. Since we are parsing the flags globally, it need not be done inside each sub-command.

Reply:
Removed the lines as part of https://github.com/openebs/node-disk-manager/pull/325/commits/3f441e29e61e631563790eb563a4d6920bd87f54